src/keys.ts

```diff
@@ -417,7 +417,7 @@ export function parsePrivateKeyEncoding(
 }
 
 function prepareSecretKey(
-  key: ArrayBuffer | KeyObject | CryptoKey | string,
+  key: ArrayBuffer | KeyObject | CryptoKey | Uint8Array | string,
```

Comment:
Can we use `RandomTypedArrays` here?  Should have said something earlier about this aggregate type.